src/texts.ts

```diff
@@ -88,15 +96,15 @@ export class Texts extends Page {
 
         if (isLinkText) {
           this.data[defaultTextToken] = {
-            ...this.getTextStyles(textStyle),
+            ...this.normalizeTextStyles(textStyle),
             color: 'text-link-accent-default',
           };
         } else if (isMobileText) {
-          this.setResponsiveTextStyles(mobileTextToken, textStyle);
+          this.setResponsiveTextStyles(mobileTextToken, textStyle, 'base');
         } else if (isDesktopText) {
-          this.setResponsiveTextStyles(desktopTextToken, textStyle);
+          this.setResponsiveTextStyles(desktopTextToken, textStyle, 'lg');
```

Comment:
is it possible to only apply this if the value is different that the mobile version?

Reply:
Added `dedupeResponsiveTextStyles` function to clean duplicated responsive values